File: backend/src/main/java/com/licenseair/backend/domainModel/SessionLogModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Long;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* SessionLog
*/
@Data()
public class SessionLogModel {
  public Long id = null;

  /**
  *
  */
  public String sign = null;

  /**
  *
  */
  public Long user_id = null;

  /**
  *
  */
  public String key = null;

  /**
  * 1 user, 2 admin
  */
  public Integer type = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}

File: generator/output/ts-model/session-log.model.ts
class SessionLogModel {
  public id: number = 0;

  // deleted 
  public deleted: boolean = false;

  // sign 
  public sign?: string;

  // user_id 
  public user_id?: number = 0;

  // key 
  public key?: string;

  // type 1 user, 2 admin
  public type?: number = 0;

  public created_at: string;

  public updated_at: string;
}

const SessionLogModelFields = {
  id: "id",
  deleted: "deleted",
  sign: "sign",
  user_id: "user_id",
  key: "key",
  type: "type",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {SessionLogModel, SessionLogModelFields};
